integration/src/test/resources/GeneralUsage.scala

```diff
@@ -62,7 +62,9 @@ object GeneralUsage {
     testAbstractExtension
     testMixedAbtractExtension
     testLiteralBoolean
-    testUnderscoreSuffix
+    testNamesWithDots
+    testNamesWithHyphens
+    testNamesWithUnderscores
```

Comment:
I wish I could have added a test case for colons but they suffered from issues #459, scala/scala-xml#94, and scala/scala-xml#182.